src/applications/Job.java

```diff
@@ -85,16 +85,15 @@ boolean moveToNextMachine(MachineShopSimulator machineShopSimulator, SimulationR
 	}
 
 	private void changeState(MachineShopSimulator machineShopSimulator, int index) {
-		if (machineShopSimulator.geteList().nextEventTime(index) == machineShopSimulator.getLargeTime()) {// machine is idle
+		if (machineShopSimulator.geteList().nextEventTime(index) == machineShopSimulator.getMaxTime()) {// machine is idle
 		    // schedule next one.
-            Job lastJob;
+            Job activeJob;
             Machine machine = machineShopSimulator.machineAt(index);
-			if (machine.getActiveJob() == null) {// in idle or change-over
-			                                            // state
-			    lastJob = null;
+			if (machine.getActiveJob() == null) {// in idle or moving to a different machine
+                activeJob = null;
 			    // wait over, ready for new job
-			    if (machine.jobQisEmpty()) // no waiting job
```

Comment:
Why remove this comment? I think that actually decreases readability, the comment does not seem excessive here.

Reply:
I think my  reasoning at the time was that there is no need for a comment since machine.jobQisEmpty is descriptive enough but yes I understand I maybe should have left the comments in.

Reply:
> Why remove this comment? I think that actually decreases readability, the comment does not seem excessive here.

I think my reasoning at the time was that there is no need for a comment since machine.jobQisEmpty is descriptive enough but yes I understand I maybe should have left the comments in.